lib/obd.js

```diff
@@ -295,12 +295,17 @@ OBDReader.prototype.connect = function (address, channel) {
 
 /**
  * Disconnects/closes the port.
+ *
+ * @param {Function} cb Callback function when the serial connection is closed
  * @this {OBDReader}
  */
-OBDReader.prototype.disconnect = function () {
+OBDReader.prototype.disconnect = function (cb) {
     clearInterval(this.intervalWriter);
     queue.length = 0; //Clears queue
-    this.btSerial.close();
+    if(typeof cb === 'function') {
+        this.btSerial.on('closed',cb) ;
+        this.btSerial.close();
```

Comment:
I rather have the bt.Serial.close outside the if check. Or am I wrong here?
Would make the callback optional.